ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3608,6 +3606,67 @@ def test_torch_instance_cumsum(
     )
 
 
+@st.composite
+def _get_clamp_inputs(draw):
+    shape = draw(
+        helpers.get_shape()
+    )
+    x_dtype, x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("float"),
```

Comment:
We should test for integers too, `torch.clamp` accepts both `float` and `int` inputs.